src/cargo/core/resolver/mod.rs

```diff
@@ -1091,12 +1088,9 @@ fn activate_deps_loop(
                             .filter_map(|(_, (deb, _, _))| {
                                 past_conflicting_activations.get(&deb).and_then(|past_bad| {
                                     // for each dependency check all of its cashed conflicts
```

Comment:
s/cashed/cached/

Reply:
Thanks spelling is **not** my strong suit! I know I'd mess this one up at some point.

Reply:
oh no worries, it's also not mine!